examples/classification/script.jl

```diff
@@ -0,0 +1,184 @@
+# # Stochastic Variational Classification
+#
+# This example demonstrates how to carry out non-conjugate Gaussian process
+# inference using the stochastic variational Gaussian process (SVGP) model. For
+# a basic introduction to the functionality of this library, please refer to the
+# [User Guide](@ref).
+#
+# ## Setup
+
+using SparseGPs
+using ParameterHandling
+using Zygote
+using Distributions
+using LinearAlgebra
+using Optim
+
+using Plots
+default(; legend=:outertopright, size=(700, 400))
+
+using Random
+Random.seed!(1234)
+#md nothing #hide
+
+# ## Generate some training data
+#
+# For our binary classification model, we will use the standard approach of a
+# latent GP with a Bernoulli likelihood. This results in a generative model that
+# we can use to produce some training data.
+#
+# First, we define the underlying latent GP
+# ```math
+# f \sim \mathcal{GP}(0, k(\cdot, \cdot'))
+# ```
+# and sample a function `f`.
+
+k_true = [30.0, 0.5]
+kernel_true = k_true[1] * (SqExponentialKernel() ∘ ScaleTransform(k_true[2]))
+
+lgp = LatentGP(GP(kernel_true), BernoulliLikelihood(), 1e-12)
+x_true = 0:0.02:6
+f_true, y_true = rand(lgp(x_true))
+
+plot(x_true, f_true; seriescolor="red", label="")  # Plot the sampled function
+
+# Then, the output of this sampled function is pushed through a logistic sigmoid
+# `μ = σ(f)` to constrain the output to `[0, 1]`.
+
+μ = mean.(lgp.lik.(f_true))
+plot(x_true, μ; seriescolor="red", label="")
+
+# Finally, the outputs `y` of the process are sampled from a Bernoulli
+# distribution with mean `μ`. We're only interested in the outputs at a subset
+# of inputs `x`, so we first pick some random input locations and then find the
+# corresponding values for `y`.
+
+N = 30  # The number of training points
+mask = sample(1:length(x_true), N; replace=false, ordered=true)  # Subsample some input locations
+x, y = x_true[mask], y_true[mask]
+
+scatter(x, y; label="Sampled outputs")
+plot!(x_true, mean.(lgp.lik.(f_true)); seriescolor="red", label="True function")
```

Comment:
```suggestion
plot!(x_true, mean.(lgp.lik.(f_true)); seriescolor="red", label="True probability")
```
or
```suggestion
plot!(x_true, mean.(lgp.lik.(f_true)); seriescolor="red", label="True mean")
```
seems more appropriate ("function" makes me think of the latent `f`)